app/javascript/app/components/ndcs/lts-explore-map/lts-explore-map-selectors.js

```diff
@@ -169,9 +169,9 @@ export const getLegend = createSelector(
 );
 
 export const getEmissionsCardData = createSelector(
-  [getLegend, getMapIndicator, getMapIndicator, getIndicatorsData],
-  (legend, indicator, selectedIndicator, indicators) => {
-    if (!indicator || !legend || !selectedIndicator) {
+  [getLegend, getMapIndicator, getIndicatorsData],
+  (legend, selectedIndicator, indicators) => {
+    if (!legend || !selectedIndicator) {
```

Comment:
Shouldn't we add the `indicators` guard here as well?
```suggestion
    if (!legend || !selectedIndicator || !indicators) {
```